### DISCUSSION TITLE
The name registered in the billing information matches the name on the student ID card, but it is rejected

### DISCUSSION THREAD

**Discussion by halnya2222** [community/community#111182] [GitHub Education]
### Hi everyone,

I am Japanese junior high school student and would like to obtain GitHub Education. However, my application is rejected, probably because the order of my name on my student ID card is different from the billing information. (On my student ID card, my name is listed in the order of my first name and last name.)

#### Reject reason
- Please ensure that your academic affiliation document contains your last name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.

- Please ensure that your academic affiliation document contains your first name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). Variations of your name and nicknames are unacceptable.

#### PS
~~Can I contact you about this issue is allowed at [GitHub Support](https://support.github.com)?~~ I recently contacted support regarding this issue.

(I use DeepL Translator, which may make some sentences unnatural or have a different meaning than intended.)

**Comment by djjangodev** [comment]
i have the same problem
